packages/commercetools/theme/nuxt.config.js

```diff
@@ -39,11 +39,14 @@ export default {
       });
     }
   },
+  plugins: [
+    './plugins/api-client.js',
```

Comment:
why you rename commercetools plugin?

Reply:
Back then I thought it was better name since it would match the part it integrates (api client) and since it's in specific integration package there is no need to name it with integration. 
Now, thanks to your comment, I realized it's being copied to destination theme in project so it's really best to keep it named according to the integration. :)